packages/cmrjs/package.json

```diff
@@ -43,6 +44,7 @@
   },
   "devDependencies": {
     "ava": "^0.25.0",
+    "fs-extra": "7.0.0",
```

Comment:
This was added as the local linter was throwing warnings as this dependency wasn't explicit and is required in the unit tests only.